src/screens/Followings/FollowersScreen.js

```diff
@@ -37,12 +41,16 @@ function FollowersScreen() {
   };
 
   React.useEffect(() => {
-    if (searchText.length === 0) {
+    if (!isFirstTimeOpen && searchText.length > 0) {
       fetchFollower(true, searchText);
     }
   }, [searchText]);
 
   React.useEffect(() => {
+    if (isFirstTimeOpen && !params?.params?.initialData) {
+      fetchFollower(true, '');
+    }
+    setIsFirstTimeOpen(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in these useEffect hooks are related to the fetching of followers. The first hook now only fetches followers when `isFirstTimeOpen` is false and `searchText.length > 0`. The second hook fetches followers on the first open if there's no initial data. This seems like a good approach to prevent unnecessary data fetching.
```javascript
React.useEffect(() => {
  if (!isFirstTimeOpen && searchText.length > 0) {
    fetchFollower(true, searchText);
  }
}, [searchText]);

React.useEffect(() => {
  if (isFirstTimeOpen && !params?.params?.initialData) {
    fetchFollower(true, '');
  }
  setIsFirstTimeOpen(false);
  setNavbarTitle('Search Users', dispatchNavbar);
}, []);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->